src/pipecat/transports/services/helpers/daily_rest.py

```diff
@@ -187,14 +284,19 @@ async def create_room(self, params: DailyRoomParams) -> DailyRoomObject:
         return room
 
     async def get_token(
-        self, room_url: str, expiry_time: float = 60 * 60, owner: bool = True
+        self,
+        room_url: str,
+        expiry_time: float = 60 * 60,
+        owner: bool = True,
+        params: DailyMeetingTokenParams = None,
```

Comment:
Should this be optional?
```suggestion
        params: Optional[DailyMeetingTokenParams] = None,
```

Reply:
Yes, i have a question here, post python3.9, general typing syntax is `DailyMeetingTokenParams|None` is there a reason we still follow the old one?